frontend/ui/widget/dictquicklookup.lua

```diff
@@ -1328,6 +1329,10 @@ function DictQuickLookup:lookupWikipedia(get_fullpage, word, is_sane, lang)
     self.ui:handleEvent(Event:new("LookupWikipedia", word, is_sane, self.word_boxes, get_fullpage, lang))
 end
 
+function DictQuickLookup:onShowMenu()
+    return self:showResultsMenu()
+end
```

Comment:
I don't know if there can be any conflict or unwanted activation of this results menu - or if by chance it would have some happy effect,, but there is already a common `ShowMenu` events triggered by Dispatcher/Gestures:
https://github.com/koreader/koreader/blob/2f0e456a428ef81de7c0747208d581f39e9359da/frontend/dispatcher.lua#L63
So, may be name this and the event `onShowResultsMenu` / `ShowResultsMenu` .